### ISSUE TITLE
License clarity

### ISSUE THREAD

**Issue by gurch101** [fin-hypergrid/core#766]
I was looking to build a React wrapper around this library but it looks like the license used for this project seems to be an amended MIT license with the following addition:

```
You may not redistribute the software or modifications as part of any
application that can be described as a development toolkit, library or
framework.
```

I Don't know the relationship between openfin/JPM but isn't JPMorgans Perspective library a violation of your license?

**Comment by joneit** [comment]
The intent was to prevent anyone from forking, merely renaming, calling it their own and possibly commercializing it as is, _i.e.,_ as a grid visualization engine.

To clarify the license, we are considering adding the above phrase "grid visualization engine" as follows:

> You may not redistribute the software or modifications as part of any application that can be described as a grid visualization engine development toolkit, library or framework.

Does that make it any clearer?

**Comment by joneit** [comment]
We also are considering "solely and specifically as a grid visualization engine" ...

**Comment by joneit** [comment]
After some internal discussion, we have decided to remove the clause in its entirety (PR #768, to be included in the `3.1.0` release).
